### PR TITLE
feat: Add suspend and resume to CronWorkflows CLI

### PR DESCRIPTION
**Pull request by simster7**
Closes: #1924

### CODE REVIEW

**Comment by simster7** [comment]
@alexec Added tests!

File: test/e2e/cron/cron_test.go [hunk 1]
```diff
@@ -27,21 +27,47 @@ func (s *CronSuite) TestBasic() {
...
+		})
+}
+
+func (s *CronSuite) TestResume() {
+	s.Given().
+		CronWorkflow("@testdata/basic.yaml").
+		When().
+		CreateCronWorkflow().
+		Then().
+		RunCli([]string{"cron", "resume", "test-cron-wf-basic"}, func(t *testing.T, output string) {
```
Comment (alexec) [inline test/e2e/cron/cron_test.go]: Interesting... perhaps we can refactor the `cli_test.go`?

File: test/e2e/fixtures/then.go [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
+	"os/exec"
 	"testing"
 
 	log "github.com/sirupsen/logrus"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	wfv1 "github.com/argoproj/argo/pkg/apis/workflow/v1alpha1"
 	"github.com/argoproj/argo/pkg/client/clientset/versioned/typed/workflow/v1alpha1"
+
+	argoexec "github.com/argoproj/pkg/exec"
```
Comment (alexec) [inline test/e2e/fixtures/then.go]: go imports

File: test/e2e/fixtures/then.go [hunk 3]
```diff
@@ -55,3 +59,17 @@ func (t *Then) ExpectWorkflowList(listOptions metav1.ListOptions, block func(*te
 	block(t.t, wfList)
 	return t
 }
+
+func (t *Then) RunCli(args []string, block func(*testing.T, string)) *Then {
+	cmd := exec.Command("../../../dist/argo", args...)
```
Comment (alexec) [inline test/e2e/fixtures/then.go]: Can you please refactor the existing `argo(...)` command?

File: test/e2e/cron/cron_test.go (hunk 1 above)
Comment (simster7) [reply test/e2e/cron/cron_test.go]: Sure, however that is still only on the `apiserverimpl` branch. Maybe we can merge this, then I'll make the changes on that branch?

File: test/e2e/fixtures/then.go (hunk 3 above)
Comment (simster7) [reply test/e2e/fixtures/then.go]: Same as https://github.com/argoproj/argo/pull/1925#discussion_r365290283

File: test/e2e/cron/cron_test.go (hunk 1 above)
Comment (alexec) [reply test/e2e/cron/cron_test.go]: ah - of course - we can merge from `master` into `apiserverimpl`

**Comment by alexec** [comment]
Ignore e2e failure.

**Pull request merged**